### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by HanaSabih**
Created method current_user. Added forms that allow for the user returned by current_user to create posts and to create comments on existing posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def create
```
Comment (rustam-tolipov) [inline app/controllers/comments_controller.rb:6]: **You did a great job!** 

- [ ] However, in your comments, I was not able to find `new` as your posts have. Kindly add new to your comments `controllers`, `routes`.

File: config/routes.rb [hunk 2]
```diff
@@ -4,7 +4,10 @@
   # Defines the root path route ("/")
   # root "articles#index"
   get "/users", to: "users#index", as: 'user'
+  get "/posts/new", to: "posts#new", as: 'posts_new'
+  post "posts", to: 'posts#create', as: 'posts_create'
   get "/users/:id", to: "users#show", as: 'users'
   get "/users/:user_id/posts", to: "posts#index", as: 'user_posts'
   get "/users/:user_id/posts/:id", to: "posts#show", as: 'user_post'
+  post 'comments/:user_id/:post_id', to: 'comments#create', as: 'comments_create'
```
Comment (rustam-tolipov) [inline config/routes.rb]: - [ ] For your comment routes should be like that:


```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new

```

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @HanaSabih,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-written readme_
_Clean code_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @HanaSabih, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to add the new action which calls the form for the comment.

## Optional suggestions

- [ ] Kindly note that one requirement was not implemented and it's very important, and this requirement is the ability for users to add likes to a post, I have added this as optional because the last reviewer did not mention it. Kindly implement this feature because it is important for your learning and future milestones.

like


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rustam-tolipov** [comment]
Hello @HanaSabih, I hope you are doing well. As @elisha2kyakpo1:

I'd like to point out that one requirement was not implemented and it's essential, and this requirement is the ability for users to add likes to a post, I have added this as optional because the last reviewer did not mention it. Kindly implement this feature because it is crucial for your learning and future milestones.

The above requirement is crucial for your upcoming project. Please, add these likes before starting the next milestone.

Sorry, for the error that I made.  If you need anything please reach me via slack.

**Pull request merged**